app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,73 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* universal */
+
+body {
+	padding-top: 60px;
```

Comment:
indent to thế e

Reply:
@manhnt-0960 
e đã chỉnh hết tất cả các ident như vậy rồi, không hiểu sao push lên vẫn thế.